js/common/objects/section.js

```diff
@@ -9,10 +9,10 @@ function Section(times, course, id) {
     'use strict';
 
     this.id = id;
-    this.courseName = this.id.substring(0, 8);
     this.name = this.id.substring(9, 14);
-    this.session = this.id.substring(15, 16);
     this.type = this.name.charAt(0);
+    this.courseName = this.id.substring(0, 6) + ' (' + this.type + ')';

```

Comment:
Thanks, I've changed it so that now the text is changed when the HTML is rendered.